backup.js

```diff
@@ -48,7 +58,17 @@ async.auto({
         callback(null, (new Date()).toISOString().split('T')[0]);
     },
 
-    getHome: function(callback) {
+    getEmergenceConfig: function(callback) {
+        fs.readFile('/emergence/config.json', 'ascii', function(err, data) {
+            if (err) {
+                return callback(err);
+            }
+
+            callback(null, JSON.parse(data));
```

Comment:
If you're not going to do any sort of error handling here you can just use ``require()`` to read JSON files.

If you do want to use ```JSON.parse``` you may want to put it in a ``try {} catch {}`` and pass back the exception if it fails to parse.

For anything user facing I think that the``jsonlint`` module will provide the line/character the error is on with a carrot. I can't remember the name.I like to use that for config files.